timesketch.conf

```diff
@@ -6,30 +6,37 @@
 #
 # $ tsctl -c /path/to/this/timesketch.conf runserver
 
-# Turn off debugging.
-# It is a security risk to have this enabled in production.
+# Show debug information.
+#
+# Note: It is a security risk to have this enabled in production.
 DEBUG = False
 
+# Key for signing cookies and for CSRF protection.
+#
 # This should be a unique random string. Don't share this with anyone.
-# The key is used for signing cookies and for CSRF protection.
 # To generate a key, you can for example use openssl:
 # $ openssl rand -base64 32
-# If the key is not set the server will not start.
 SECRET_KEY = u''
```

Comment:
Yep, this is redundant because the server will complain in any case if the key is missing and refuse to start.